scripts/docker-compose.yml

```diff
@@ -37,7 +37,7 @@ services:
       - ../services/traction:/app:rw
 
   traction-agent:
-    image: bcgovimages/aries-cloudagent:py36-1.16-1_0.7.2
+    image: bcgovimages/aries-cloudagent:py36-1.16-1_0.7.3
```

Comment:
Need to make the same change in the charts: endorser/values.yaml, traction/values.yaml

Reply:
ok thanks!